.github/workflows/build.yml

```diff
@@ -6,21 +6,16 @@ jobs:
   build:
     name: Build mathlib
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        lean_version:
-          - leanprover-community/lean:3.5.1
-      fail-fast: false
     steps:
       - uses: actions/checkout@v2
 
       - name: install elan
         run: |
           set -o pipefail
           curl https://raw.githubusercontent.com/Kha/elan/master/elan-init.sh -sSf | sh -s -- --default-toolchain none -y
-          ~/.elan/bin/elan override set ${{ matrix.lean_version }}
           ~/.elan/bin/lean --version
           echo "::add-path::$HOME/.elan/bin"
+          short_lean_version="$(~/.elan/bin/lean --run scripts/lean_version.lean)"
```

Comment:
This line here has no effect.  Each step is executed in a separate shell.

Reply:
Ah, you're right. I think this is the way to store it permanently:
```
echo "::set-env name=short_lean_version::$(~/.elan/bin/lean --run scripts/lean_version.lean)"
```